### PR TITLE
Be able to get last updated time from views and materialized views 

### PR DESCRIPTION
**Pull request by dgaubert**
Even if they come from remotes databases

### CODE REVIEW

File: lib/querytables.js [hunk 1]
```diff
@@ -175,14 +175,67 @@ function getForeignTableUpdatedAt (pg, table, callback) {
...
+async function setUpdateAtToNonRegularTables (pg, rows) {
     const getForeignTableUpdatedAtPromise = promisify(getForeignTableUpdatedAt);
+
     for (const row of rows) {
-        if (row.relkind === 'f') {
+        switch (row.relkind) {
+        case 'f': // foreign table
             row.updated_at = await getForeignTableUpdatedAtPromise(pg, row);
+            break;
+        case 'm': // materialized view
```
Comment (Algunenano) [inline lib/querytables.js]: I don't agree with this design decision, materialized view should be treated as regular tables to manage cache. Changes in the source table don't matter at all as long as you refresh the materialized view, so invalidating the cache won't mean anything. IMO, materialized views should have an entry in the local table updated when they are refreshed.

Comment (dgaubert) [reply lib/querytables.js]: It needs some analysis to understand what is the best approach. Going to the definition to know when it was updated doesn't make sense if the materialized view was not refreshed. We need to implement a way to save when a materialized view was updated.

Comment (dgaubert) [reply lib/querytables.js]: I'm going to revert this part as I'd like to move this forward and reevaluate it when we have a clear idea of what we need with Federated tables. Do you agree?

Comment (Algunenano) [reply lib/querytables.js]: Absolutely.

**Pull request merged**